build.assets/version-ops/cmd/check/main_test.go

```diff
@@ -13,11 +13,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
 package main
 
 import (
 	"context"
 	"testing"
+
+	"github.com/google/go-github/v41/github"
```

Comment:
If you're up for including it, the rest of teleport is using Testify for tests (`github.com/stretchr/testify`), which may simplify your test  cases a bit.

```go
tests := []struct {
		desc     string
		tag      string
		releases []string
		wantErr  require.ErrorAssertionFunc
	}{
		{
			desc:    "fail-rc",
			tag:     "v9.0.0-rc.1",
			wantErr: require.Error
		},
		//...
		{
			desc:    "pass",
			tag:     "v8.0.1",
			wantErr: require.NoError,
		},
	}
	for _, test := range tests {
		t.Run(test.desc, func(t *testing.T) {
			test.wantError(t, checkPrerelease(test.tag))
		})
	}
}
```

Not a huge win for one test, but it makes everything have the same vocabulary

Reply:
[2nd mention](https://github.com/gravitational/teleport/pull/10295#discussion_r805071270) is a charm. :smile:  Done in 6e10e88.